atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -238,8 +239,25 @@ def _make_labels(self):
 
     def do_worker_build(self, release, cluster_info):
         build = None
+        task_id = None
+        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
+        if fs_result is not None:
+            try:
+                fs_task_id = fs_result['filesystem-koji-task-id']
+            except KeyError:
+                self.log.error("%s: expected filesystem-koji-task-id in result",
+                               AddFilesystemPlugin.key)
+            else:
+                try:
+                    task_id = int(fs_task_id)
+                except ValueError:
+                    self.log.error("invalid taskID %r", fs_task_id, exc_info=1)
+
         try:
             kwargs = self.get_worker_build_kwargs(release, cluster_info.platform)
+            self.log.debug("%s: setting filesystem_koji_task_id to %d",
+                           AddFilesystemPlugin.key, task_id)
+            kwargs['filesystem_koji_task_id'] = task_id
```

Comment:
By truthy I mean a value that evaluates to true in an expression, which should handle not passing the parameter if the value is None for example. 
A simple `if task_id` would do exactly that.